app/src/components/features/rules/RulePairs/Filters/index.jsx

```diff
@@ -375,7 +379,11 @@ const Filters = (props) => {
                 dangerLight: "#323337", // tag cancel background color
               },
             })}
-            options={RESOURCE_TYPE_OPTIONS}
+            options={
+              !isScriptRule
+                ? RESOURCE_TYPE_OPTIONS
+                : RESOURCE_TYPE_OPTIONS.filter((option) => option.value === "main_frame")
```

Comment:
We should support only Iframe too

Reply:
<img width="951" alt="Snipaste_2024-08-06_10-29-17" src="https://github.com/user-attachments/assets/5016db9c-7613-49d8-9f08-0d6b2317dafd">